### PR TITLE
Incorporating the schedule step as well as subscribing to the pose topic

### PR DESCRIPTION
**Pull request by knord94**

### CODE REVIEW

**Comment by oysand** [comment]
Run black for formatting

File: src/isar_turtlebot/robotinterface.py [hunk 1]
```diff
@@ -27,36 +34,45 @@ def __init__(self):
...
-                        "header": {
-                            "seq": 0,
-                            "stamp": {"secs": 1533, "nsecs": 746000000},
-                            "frame_id": "map",
+            self.publish_navigation_task(pose=step.pose)
+
+        elif isinstance(step, (TakeImage, TakeThermalImage)):
+            raise TypeError(f"Image step is not  yet implemented on turtlebot")
+
+    def publish_navigation_task(self, pose: Pose) -> None:
```
Comment (oysand) [inline src/isar_turtlebot/robotinterface.py:49]: Is this beyond the scope of the commit? If so, consider splitting the commit or rewording it to incorporating the schedule step as well as subscribing to the pose topic.

Comment (knord94) [reply src/isar_turtlebot/robotinterface.py:49]: Changed the commit message accordingly

**Review by oysand — APPROVED** [review]
LGTM

**Pull request merged**